VirtoCommerce.CustomerModule.Web/Scripts/blades/member-list.js

```diff
@@ -130,7 +130,19 @@ function ($scope, members, dialogService, bladeUtils, uiGridHelper, memberTypesR
                     bladeNavigationService.closeChildrenBlades(blade, function () {
                         var memberIds = _.pluck(selection, 'id');
 
-                        if (_.any(memberIds)) {
+                        if (($scope.gridApi != undefined) && $scope.gridApi.selection.getSelectAllState()) {
```

Comment:
Can we use current search criteria object and pass it to the API? Instead of creating new one each time?

Reply:
If save the criteria to variable then sometimes the API get a empty  criteria.